src/index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
-    <div id="map">Map goes here?</div>
+    <div id="map"></div>
+    <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What's this for?

Reply:
yup line deleted, was obsolete code from the nzsvg website